### PR TITLE
Plasma fire sounds

### PR DESCRIPTION
**Pull request by ghost**
🆑
soundadd: You can now hear plasma fires.
/🆑

Ported from https://github.com/yogstation13/yogstation/pull/2498
see that for more info

### CODE REVIEW

**Comment by MoreRobustThanYou** [comment]
I just tested, it works, and sounds good.

**Comment by kevinz000** [comment]
every hotspot calls this on process
I /SERIOUSLY/ hope you know what you're doing.

File: code/modules/atmospherics/environmental/LINDA_fire.dm [hunk 1]
```diff
@@ -144,6 +147,9 @@
 		/*if(prob(25))
 			location.ReplaceWithSpace()
 			return 0*/
+	if(world.time - sound_start_time >= sound_length) //looping sound
```
Comment (Cyberboss) [inline code/modules/atmospherics/environmental/LINDA_fire.dm]: Use a LOW prob instead

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by MoreRobustThanYou** [comment]
@kevinz000 i didn't make the code, this was ported from a yog pr

File: code/modules/atmospherics/environmental/LINDA_fire.dm (hunk 1 above)
Comment (RemieRichards) [reply code/modules/atmospherics/environmental/LINDA_fire.dm]: the point is to repeat the sound once it finishes, cyberboss
I'm thinking using the ```repeat``` var of sound datums would be better (should be entirely client side cost) but would probably need a dedicated sound channel

**Comment by RemieRichards** [comment]
@MoreRobustThanYou as you are the one proposing to bring it here, it is up to you to ensure it is up to standard.

**Review by RemieRichards — CHANGES_REQUESTED** [review]
Make it a client side loop
(the ```repeat``` var on sounds)

**Comment by ma44** [comment]
inb4 crashes server

**Comment by Qustinnus** [comment]
@MoreRobustThanYou that just makes it worse because that means Matsku made it.


spoiler: matsku does not know what he's doing.

**Comment by BeeSting12** [comment]
do we really need an obnoxious sound to go with plasmafires?

**Comment by ShizCalev** [comment]
firealarm spam 2.0

**Comment by WJohn** [comment]
See though, this isn't ear rape like the fire alarm.

Speaking of which, when the hell are we reverting the fire alarm's sound to something that isn't ear rape?

**Pull request closed without merging**